x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_single_domain_logic.test.ts

```diff
@@ -161,6 +171,7 @@ describe('CrawlerSingleDomainLogic', () => {
           '/internal/app_search/engines/some-engine/crawler/domains/1234'
         );
 
+        expect(CrawlerLogic.actions.fetchCrawlerData).toHaveBeenCalled();
```

Comment:
_This_ is why domains weren't being removed from the Overview page after a domain was deleted from its own Single Domain page